frontend/sw360-portlet/src/main/webapp/css/sw360.css

```diff
@@ -326,3 +326,10 @@ input[readonly].clickable {
     word-break: break-all;
     font-family: monospace;
 }
+
+.formatSelect {
+   width: 20em !important;
```

Comment:
The relevant SW360.css is imported last anyway. I tried to put it up an move it directly after the last .css import, but that doesn't do the trick. Maybe we should have a look together or leave it as it is.